init_suite.py

```diff
@@ -34,10 +34,13 @@ def process_override(dir_name: str) -> List:
     The directory must contain two files if you want to override.
         - <test_suite>.yaml
         - overrides.yaml
-
+    The directory may contain more than one test suite file and their order of execution is in ascending order of filenames
```

Comment:
Please move to the next line once you cross 110 characters

Reply:
ok pragadeesh